src/python_testing/TC_DeviceConformance.py

```diff
@@ -236,6 +238,51 @@ def test_TC_IDM_10_3(self):
         if not success:
             self.fail_current_test("Problems with cluster revision on at least one cluster")
 
+    def test_IDM_10_5(self):
+        success = True
+        warn_on_extra_clusters = self.user_params.get("warn_on_extra_clusters", True)
+        test_name = self.get_test_name()
+
+        for endpoint_id, endpoint in self.endpoints.items():
+            device_types = [x for x in endpoint[Clusters.Descriptor]
+                            [Clusters.Descriptor.Attributes.DeviceTypeList] if x.deviceType <= 0xBFFF]
+            endpoint_clusters = []
+            for device_type in device_types:
+                device_type_id = device_type.deviceType
+                if device_type_id not in self.xml_device_types.keys():
+                    # TODO: report a problem here
```

Comment:
why is this still a todo? could we explain why it is not done here and why we are waiting for? myabe a github issue for incomplete bits (same for revision check below)